gtests/net/mptcp/remove_addr/remove_addr_server_v4.pkt

```diff
@@ -0,0 +1,28 @@
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0     `ip mptcp endpoint flush`
++0     `ip mptcp endpoint add 198.51.100.2 id 10 signal`
```

Comment:
if we call a script that checks if we are in v4/v6, could we merge both v4 and v6 script?

Reply:
good point. As long as we don't care about matching the value of the address in the `ADD_ADDR`, that should be do-able. Ideally, it would be nice to to "parametrize" the script so that the number of  endpoints is variable: that would allow also writing a script that adds coverage for multi-id `RM_ADDR` (that seems to be supported by the parser, so it would deserve some coverage as well). @matttbe, WDYT?

Reply:
yes, I thought about the address but on the other hand, I don't think we check it elsewhere (add_addr). Do we?
If we do, maybe OK to have this small script dedicated to one or the others, at least we check that :-)

If we need to validate more advanced situations, we would use `common/server.sh` I guess

Reply:
> if we call a script that checks if we are in v4/v6, could we merge both v4 and v6 script?

with a fix similar done for issue #143 , we can test v4 and v6 using the same script.